nservicebus/azure-service-bus/dlq-forwarding.md

```diff
@@ -0,0 +1,28 @@
+---
+title: Azure Service Bus DLQ Forwarding
+versions: '[7,)'
+reviewed: 2016-08-29
+tags:
+- Azure
+- Cloud
+- Error Handling
+---
+
+## Dead Letter Queue Forwarding
+
+When Azure Service Bus service detects that an endpoint cannot process messages, or when it has been configured not to discard expired or filtered messages, then it will move those messages to a dead letter queue associated with the entity, originally contained those messages.
+
+From a management perspective, this is less than ideal when operations staff has to monitor a multitude of entities for dead lettered or discarded messages.
```

Comment:
>From a management perspective, this is less than ideal when operations staff has to monitor a multitude of entities for dead lettered or discarded messages.

When you mention this is less than ideal, it's not very clear as to what "this" refers to.

Do you mean:
From a management perspective, it is not ideal for an operations staff to monitor a multitude of endpoints for dead lettered or discarded messages in multiple locations.